includes/User.php

```diff
@@ -1356,7 +1356,7 @@ public static function getDefaultOption( $opt ) {
 	 *                    done against master.
 	 * @param $shouldLogBlockInStats Bool flag that decides whether to log or not in PhalanxStats
 	 */
-	private function getBlockedStatus( $bFromSlave = true, $shouldLogBlockInStats = true ) {
+	private function getBlockedStatus( $bFromSlave = true, $shouldLogBlockInStats = true, $global = true ) {
```

Comment:
:speech_balloon: — for backwards compatibility, assume we are dealing with a global block.